flutter/cpp/binary/main.cc

```diff
@@ -370,7 +429,14 @@ int Main(int argc, char *argv[]) {
                       batch_size);
   driver.RunMLPerfTest(mode, min_query_count, min_duration / 1000.0,
                        single_stream_expected_latency_ns, output_dir);
-  LOG(INFO) << "Accuracy: " << driver.ComputeAccuracyString();
+
+  if (driver.HasAccuracy()) {
+    // Report accuracy if computed
+    report_accuracy(results_file,benchmark_id,driver.ComputeAccuracyString());
+  } else {
+    // Report performance
+    report_performance(output_dir+"\\mlperf_log_summary.txt",results_file,benchmark_id, scenario == "Offline");
+  }
```

Comment:
I am not sure if I understand the condition `driver.HasAccuracy()` here. Anyway if I have a valid `groundtruth_file` or `ground_truth_directory` then `driver.HasAccuracy()` is always true and `report_performance()` is never called.

Reply:
I want to generate only accuracy results for accuracy run and only performance results for performance runs. For performance runs, we will pass blank groundtruth_file from the command line, helping with the accuracy.

Reply:
Can we check for the `mode {PerformanceOnly, AccuracyOnly, SubmissionRun}` instead and call the `report_*()` function accordingly?